### ISSUE TITLE
全球30篇最热门的计算机视觉和深度学习论文

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/datanmachlearn#12]
# 全球30篇最热门的计算机视觉和深度学习论文
- Author: [http://www.afenxi.com](http://www.afenxi.com)
- Origin: [http://www.afenxi.com/post/19793](http://www.afenxi.com/post/19793)
- Related:

> 1.引用次数：5518
> 标题：深度卷积神经网络分类(Imagenet classification with deep convolutional neural networks)
> A Krizhevsky, I Sutskever, GE Hinton, 2012
> [论文链接地址](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf?spm=5176.100239.blogcont55892.5.pm8zm1&file=4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf)
> 
> 2.引用次数：1868
> 标题：咖啡：快速卷积结构特征嵌入(Caffe: Convolutional architecture for fast feature embedding)
> Y Jia, E Shelhamer, J Donahue, S Karayev…, 2014
> [论文链接地址](http://dl.acm.org/citation.cfm?spm=5176.100239.blogcont55892.6.pm8zm1&id=2654889)
> 
> 3.引用次数：1681
> 标题：反向传播算法在手写体邮政编码识别中的应用(Backpropagation applied to handwritten zip code recognition)
> Y LeCun, B Boser, JS Denker, D Henderson…, 1989
> [论文链接地址](http://www.mitpressjournals.org/doi/abs/10.1162/neco.1989.1.4.541?spm=5176.100239.blogcont55892.7.pm8zm1#.V2fy_-wQgg8)
> 
> 4.引用次数：1516
> 标题：实现精确对象和语义段检测的富特征层次结构(Rich feature hierarchies for accurate object detection and semantic segment)
> R Girshick, J Donahue, T Darrell…, 2014
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2014/papers/Girshick_Rich_Feature_Hierarchies_2014_CVPR_paper.pdf?spm=5176.100239.blogcont55892.8.pm8zm1&file=Girshick_Rich_Feature_Hierarchies_2014_CVPR_paper.pdf)
> 
> 5.引用次数：1405
> 标题： 进行大型图像识别的深层网络(Very deep convolutional networks for large-scale image recognition)
> K Simonyan, A Zisserman, 2014
> [论文链接地址](http://arxiv.org/abs/1409.1556?spm=5176.100239.blogcont55892.9.pm8zm1&file=1409.1556)
> 
> 6.引用次数：1169
> 标题：通过共适应的特征探测器来改进神经网络(Improving neural networks by preventing co-adaptation of feature detectors)
> GE Hinton, N Srivastava, A Krizhevsky…, 2012
> [论文链接地址](http://arxiv.org/abs/1207.0580?spm=5176.100239.blogcont55892.10.pm8zm1&file=1207.0580)
> 
> 7.引用次数：1160
> 标题：深入了解卷积(Going deeper with convolutions)
> C Szegedy, W Liu, Y Jia, P Sermanet…, 2015
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Szegedy_Going_Deeper_With_2015_CVPR_paper.pdf?spm=5176.100239.blogcont55892.11.pm8zm1&file=Szegedy_Going_Deeper_With_2015_CVPR_paper.pdf)
> 
> 8.引用次数：977
> 标题：反向传播网络的手写数字识别(Handwritten digit recognition with a back-propagation network)
> BB Le Cun, JS Denker, D Henderson…, 1990
> [论文链接地址](http://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=CB3ADCB9B99C266A5ECA1B7FA92D3AC7?spm=5176.100239.blogcont55892.12.pm8zm1&doi=10.1.1.32.5076&rep=rep1&type=pdf)
> 
> 9.引用次数：907
> 标题：观察和理解卷积网络(Visualizing and understanding convolutional networks)
> MD Zeiler, R Fergus, 2014
> [论文链接地址](http://link.springer.com/chapter/10.1007/978-3-319-10590-1_53?spm=5176.100239.blogcont55892.13.pm8zm1)
> 
> 10.引用次数：839
> 标题： 降阶—以简单的方法来防止神经网络拟合(Dropout: a simple way to prevent neural networks from overfitting)
> N Srivastava, GE Hinton, A Krizhevsky…, 2014
> [论文链接地址](http://www.jmlr.org/papers/volume15/srivastava14a.old/source/srivastava14a.pdf?spm=5176.100239.blogcont55892.14.pm8zm1&file=srivastava14a.pdf)
> 
> 11.引用次数：839
> 标题：Overfeat框架:使用卷积网络进行综合识别、 定位和检测(Overfeat: Integrated recognition, localization and detection using convolutional networks)
> P Sermanet, D Eigen, X Zhang, M Mathieu…, 2013
> [论文链接地址](http://arxiv.org/abs/1312.6229?spm=5176.100239.blogcont55892.15.pm8zm1&file=1312.6229)
> 
> 12.引用次数：818
> 标题：从微图像里学习多层次特征(Learning multiple layers of features from tiny images)
> A Krizhevsky, G Hinton, 2009
> [论文链接地址](http://citeseerx.ist.psu.edu/viewdoc/download?spm=5176.100239.blogcont55892.16.pm8zm1&doi=10.1.1.222.9220&rep=rep1&type=pdf)
> 
> 13.引用次数：718
> 标题：无咖啡因:一个用于通用图像识别的深积激活特征(DeCAF: A Deep Convolutional Activation Feature for Generic Visual Recognition)
> J Donahue, Y Jia, O Vinyals, J Hoffman, N Zhang…, 2014
> [论文链接地址](http://www.jmlr.org/proceedings/papers/v32/donahue14.pdf?spm=5176.100239.blogcont55892.17.pm8zm1&file=donahue14.pdf)
> 
> 14.引用次数：691
> 标题： 深度人脸:拉近与真人之间的面部识别差距(Deepface: Closing the gap to human-level performance in face verification)
> Y Taigman, M Yang, MA Ranzato…, 2014
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2014/papers/Taigman_DeepFace_Closing_the_2014_CVPR_paper.pdf?spm=5176.100239.blogcont55892.18.pm8zm1&file=Taigman_DeepFace_Closing_the_2014_CVPR_paper.pdf)
> 
> 15.引用次数：679
> 标题：深入玻耳兹曼机 (Deep Boltzmann Machines)
> R Salakhutdinov, GE Hinton, 2009
> [论文链接地址](http://www.jmlr.org/proceedings/papers/v5/salakhutdinov09a/salakhutdinov09a.pdf?spm=5176.100239.blogcont55892.19.pm8zm1&file=salakhutdinov09a.pdf)
> 
> 16.引用次数：670
> 标题：用于图像、 语音和时间序列的卷积网络(Convolutional networks for images, speech, and time series)
> Y LeCun, Y Bengio, 1995
> [论文链接地址](https://www.researchgate.net/publication/2453996_Convolutional_Networks_for_Images_Speech_and_Time-Series?spm=5176.100239.blogcont55892.20.pm8zm1)
> 
> 17.引用次数：570
> 标题：现成的 CNN 功能: 令人震惊的基线识别(CNN features off-the-shelf: an astounding baseline for recognition)
> A Sharif Razavian, H Azizpour, J Sullivan…, 2014
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_workshops_2014/W15/papers/Razavian_CNN_Features_Off-the-Shelf_2014_CVPR_paper.pdf?spm=5176.100239.blogcont55892.21.pm8zm1&file=Razavian_CNN_Features_Off-the-Shelf_2014_CVPR_paper.pdf)
> 
> 18.引用次数：549
> 标题：学习场景标识中的分层结构特征 (Learning hierarchical features for scene labeling)
> C Farabet, C Couprie, L Najman…, 2013
> [论文链接地址](http://ieeexplore.ieee.org/xpl/login.jsp?spm=5176.100239.blogcont55892.22.pm8zm1&tp=&arnumber=6338939&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6338939)
> 
> 19.引用次数：510
> 标题：完全卷积网络的语义分割(Fully convolutional networks for semantic segmentation)
> J Long, E Shelhamer, T Darrell, 2015
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Long_Fully_Convolutional_Networks_2015_CVPR_paper.pdf?spm=5176.100239.blogcont55892.23.pm8zm1&file=Long_Fully_Convolutional_Networks_2015_CVPR_paper.pdf)
> 
> 20.引用次数：469
> 标题： 最大输出网络(Maxout networks)
> IJ Goodfellow, D Warde-Farley, M Mirza, AC Courville…, 2013
> [论文链接地址](http://www.jmlr.org/proceedings/papers/v28/goodfellow13.pdf?spm=5176.100239.blogcont55892.24.pm8zm1&file=goodfellow13.pdf)
> 
> 21.引用次数：453
> 标题：精绝的细节:深度钻研卷积网(Return of the devil in the details: Delving deep into convolutional nets)
> K Chatfield, K Simonyan, A Vedaldi…, 2014
> [论文链接地址](http://arxiv.org/abs/1405.3531?spm=5176.100239.blogcont55892.25.pm8zm1&file=1405.3531)
> 
> 22.引用次数：445
> 标题：使用卷积神经网络进行大型视频分类 (Large-scale video classification with convolutional neural networks)
> A Karpathy, G Toderici, S Shetty, T Leung…, 2014
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2014/papers/Karpathy_Large-scale_Video_Classification_2014_CVPR_paper.pdf?spm=5176.100239.blogcont55892.26.pm8zm1&file=Karpathy_Large-scale_Video_Classification_2014_CVPR_paper.pdf)
> 
> 23.引用次数：347
> 标题：深度视觉语义路线生成图像描述(Deep visual-semantic alignments for generating image descriptions)
> A Karpathy, L Fei-Fei, 2015
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Karpathy_Deep_Visual-Semantic_Alignments_2015_CVPR_paper.pdf?spm=5176.100239.blogcont55892.27.pm8zm1&file=Karpathy_Deep_Visual-Semantic_Alignments_2015_CVPR_paper.pdf)
> 
> 24.引用次数：342
> 标题：深入学习整流器:超越人类水平的图像分类(Delving deep into rectifiers: Surpassing human-level performance on imagenet classification)
> K He, X Zhang, S Ren, J Sun, 2015
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_iccv_2015/papers/He_Delving_Deep_into_ICCV_2015_paper.pdf?spm=5176.100239.blogcont55892.28.pm8zm1&file=He_Delving_Deep_into_ICCV_2015_paper.pdf)
> 
> 25.引用次数：334
> 标题：使用卷积神经网络来学习和转移中级图像显示 (Learning and transferring mid-level image representations using convolutional neural networks)
> M Oquab, L Bottou, I Laptev, J Sivic, 2014
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2014/papers/Oquab_Learning_and_Transferring_2014_CVPR_paper.pdf?spm=5176.100239.blogcont55892.29.pm8zm1&file=Oquab_Learning_and_Transferring_2014_CVPR_paper.pdf)
> 
> 26.引用次数：333
> 标题：卷积网络及其在视觉中的应用(Convolutional networks and applications in vision)
> Y LeCun, K Kavukcuoglu, C Farabet, 2010
> [论文链接地址](http://research2.fit.edu/ice/sites/default/files/Convolutional%20networks%20and%20applications%20in%20vision_0.pdf?spm=5176.100239.blogcont55892.30.pm8zm1&file=Convolutional%20networks%20and%20applications%20in%20vision_0.pdf)
> 
> 27.引用次数：332
> 标题：使用场景数据库进行深度场景识别(Learning deep features for scene recognition using places database)
> B Zhou, A Lapedriza, J Xiao, A Torralba…,2014
> [论文链接地址](http://papers.nips.cc/paper/5349-learning-deep-features-for-scene-recognition-using-places-database.pdf?spm=5176.100239.blogcont55892.31.pm8zm1&file=5349-learning-deep-features-for-scene-recognition-using-places-database.pdf)
> 
> 28.引用次数：299
> 标题：使用深积网络中的空间金字塔池进行视觉识别 (Spatial pyramid pooling in deep convolutional networks for visual recognition)
> K He, X Zhang, S Ren, J Sun, 2014
> [论文链接地址](http://link.springer.com/chapter/10.1007/978-3-319-10578-9_23?spm=5176.100239.blogcont55892.32.pm8zm1)
> 
> 29.引用次数：268
> 标题：使用长期递归卷积网络进行视觉识别和描述(Long-term recurrent convolutional networks for visual recognition and description)
> J Donahue, L Anne Hendricks…, 2015
> [论文链接地址](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Donahue_Long-Term_Recurrent_Convolutional_2015_CVPR_paper.pdf?spm=5176.100239.blogcont55892.33.pm8zm1&file=Donahue_Long-Term_Recurrent_Convolutional_2015_CVPR_paper.pdf)
> 
> 30.引用次数：261
> 标题：使用二流卷积网络进行动作视频识别(Two-stream convolutional networks for action recognition in videos)
> K Simonyan, A Zisserman, 2014
> [论文链接地址](http://papers.nips.cc/paper/5353-two-stream-convolutional-networks-for-action-recognition-in-videos.pdf?spm=5176.100239.blogcont55892.34.pm8zm1&file=5353-two-stream-convolutional-networks-for-action-recognition-in-videos.pdf)
